### DISCUSSION TITLE
Client status shows Connecting, "connecting: Wait"

### DISCUSSION THREAD

**Discussion by gr8hax** [angristan/openvpn-install#870] [Help]
After using either the auto headless or the user input install.  after importing the client.ovpn to the openvpn client.

Trying to connect to the openvpn server, it tries to connect without succeeding.

it seems to fails on the "TLS Error: TLS key negotiation failed to occur within 60 seconds (check your network connectivity)."
"TLS Error: TLS handshake failed"

I know that the internet is working on both ends.

There are additional logs show that UDPv4 link local: not bound)"

SIGUSR1 [soft,tlserror] received,process restarting

I am running manjaro (latest version) on a VPS.

Please advise.

Thank you.

**Comment by TinCanTech** [comment]
Please See your server log.

**Comment by gr8hax** [comment]
The logs from the client or the logs from the install form your github
clone?  If it is from the clone, where can I find it?
